### ISSUE TITLE
Failed certbot renewal after server migration

### ISSUE THREAD

**Issue by jernst** [uboslinux/ubos-admin#879]
```
TRACE: slurpFile( /etc/ubos/config.json )
INFO : Renewing LetsEncrypt certificate(s)
TRACE: Exec: ( TERM=dumb certbot renew --quiet --agree-tos --max-log-backups 0 ) >/tmp/GZm5wb_JY_ 2>&1
TRACE: slurpFile( /tmp/GZm5wb_JY_ )
WARN : Renewing certificates from LetsEncrypt failed:
Failed to renew certificate <site> with error: Account at /etc/letsencrypt/accounts/acme-v01.api.letsencrypt.org/directory/6bf0fdc8237f055b6f18bdfd4e53781d does not exist
All renewals failed. The following certificates could not be renewed:
  /etc/letsencrypt/live/<site>/fullchain.pem (failure)
1 renew failure(s), 0 parse failure(s)
```

**Comment by jernst** [comment]
Related. When re-deploying with site JSON without the key:
```
ERROR: Cannot read LetsEncrypt live (2) key and crt file: <undef> <undef>
Use of uninitialized value $keyFile in concatenation (.) or string at /usr/share/perl5/vendor_perl/UBOS/Roles/apache2.pm line 422.
Use of uninitialized value $crtFile in concatenation (.) or string at /usr/share/perl5/vendor_perl/UBOS/Roles/apache2.pm line 422.
ERROR: httpd.service reload failed: Job for httpd.service failed.
```

**Comment by jernst** [comment]
Fixed in 05e8c7cce74817a24aad39ed14b0f8639b6d2ba1
